sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/EncryptionCosmosAsyncContainer.java

```diff
@@ -565,8 +597,11 @@ ItemDeserializer getItemDeserializer() {
         );
     }
 
-    private <T> Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> queryDecryptionTransformer(Class<T> classType,
-                                                                                                   Function<CosmosPagedFluxOptions, Flux<FeedResponse<JsonNode>>> func) {
+    private <T> Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> queryDecryptionTransformer(
+        Class<T> classType,
+        Function<CosmosPagedFluxOptions, Flux<FeedResponse<JsonNode>>> func,
+        boolean useEtagAsContinuation) {
```

Comment:
useEtagAsContinuation... why etag?

Reply:
Flag is used in FeedResponse - for Changefeed responses from the backend the continuation isn't passed as header but instead the etag is used. This flag needed to be passed through to be able to recreate a new FeedResponse in the PagedFlized etc. from a list of results of <T>